Makefile

```diff
@@ -139,17 +139,41 @@ regen-local-nix-derivations:
 check-local-nix-derivations: regen-local-nix-derivations
 	git diff --exit-code
 
-# reset db using cabal
-.PHONY: db-reset-package
-db-reset-package: c
+# Usage:
+#
+# Reset all keyspaces
+# make db-reset
+#
+# Reset keyspace for only one service, say galley:
+# make db-reset package=galley
+.PHONY: db-reset
+db-reset: c
+ifeq ($(package), all)
+	./dist/brig-schema --keyspace brig_test --replication-factor 1 --reset
+	./dist/galley-schema --keyspace galley_test --replication-factor 1 --reset
+	./dist/gundeck-schema --keyspace gundeck_test --replication-factor 1 --reset
+	./dist/spar-schema --keyspace spar_test --replication-factor 1 --reset
+else
 	$(EXE_SCHEMA) --keyspace $(package)_test --replication-factor 1 --reset
+endif
 
-# migrate db using cabal
-# For using stack see the Makefile of the package, e.g. services/brig/Makefile
-# Usage: make db-migrate-package package=galley
-.PHONY: db-migrate-package
-db-migrate-package: c
```

Comment:
can you keep a goal that juts emits a deprecation warning tell you what to call instead?  (for both reset and migrate.)